Sasquatch/SasquatchObjC/AppDelegate.m

```diff
@@ -268,12 +268,13 @@ - (void)analytics:(MSAnalytics *)analytics didFailSendingEventLog:(MSEventLog *)
 #pragma mark - MSRemoteOperationDelegate
 
 - (void)data:(MSData *)data
-    didCompletePendingOperation:(NSString *)operation
-                    forDocument:(MSDocumentWrapper *_Nullable)document
-                      withError:(MSDataError *_Nullable)error {
+    didCompleteRemoteOperation:(NSString *)operation
+           forDocumentMetadata:(MSDocumentMetadata *_Nullable)documentMetadata
+                     withError:(MSDataError *_Nullable)error {
   NSLog(@"Operation processed: %@ ", operation);
-  if (document) {
-    NSLog(@"Document: Partition : %@, document id : %@, eTag : %@ ", document.partition, document.documentId, document.eTag);
+  if (documentMetadata) {
```

Comment:
We should add this to the swift app too in AppDelegate.swift I think it was a miss before.

Reply:
I searched for all reference to the old method name didCompletePendingOperation and theyare all gone.